### PR TITLE
Update to freshest spine

### PR DESCRIPTION
**Pull request by dmdashenkov**
In this PR we update the example project to the freshest Spine API.

### CODE REVIEW

**Comment by dmdashenkov** [comment]
@armiol, PTAL.

**Comment by dmdashenkov** [comment]
@dmytro-grankin, PTAL.

File: api-java/src/test/java/io/spine/examples/todolist/c/aggregate/LabelAggregateTest.java [hunk 1]
```diff
@@ -102,17 +102,17 @@ public void setUp() {
             createBasicLabel();
         }
 
-        private List<? extends Message> dispatchUpdateLabelDetails(UpdateLabelDetails details) {
+        private List<? extends Message> dispatchCommandUpdateLabelDetails(UpdateLabelDetails details) {
```
Comment (dmytro-grankin) [inline api-java/src/test/java/io/spine/examples/todolist/c/aggregate/LabelAggregateTest.java]: `dispatchUpdateLabelDetailsCommand` looks more readable than `dispatchCommandUpdateLabelDetails`. Please apply to the same cases.

File: api-java/src/test/java/io/spine/examples/todolist/c/aggregate/definition/CompleteTaskTest.java [hunk 2]
```diff
@@ -62,9 +62,9 @@ protected void setUp() {
     @Test
     @DisplayName("produce TaskCompleted event")
     void produceEvent() {
-        dispatchCreateTaskCmd();
+        dispatchCommandCreateTaskCmd();
```
Comment (dmytro-grankin) [inline api-java/src/test/java/io/spine/examples/todolist/c/aggregate/definition/CompleteTaskTest.java]: Please remove `Cmd` part from the function name. There are a lot of similar cases below.

**Comment by dmdashenkov** [comment]
@dmytro-grankin, PTAL again.

**Pull request merged**